docs/xdebug.md

```diff
@@ -1,75 +1,28 @@
-# Installing Xdebug
+# Using Xdebug
 
-The default Docker stack is shipped without [Xdebug](https://xdebug.org/),
+The default development image is shipped with [Xdebug](https://xdebug.org/),
 a popular debugger and profiler for PHP.
-It's easy, though, to add it to your project.
 
-## Add a Debug Stage to the Dockerfile
+Because it has a significant performance overhead, Xdebug is disabled by default.
+It can be enabled by setting the `XDEBUG_MODE` environment variable to `debug`.
 
-To avoid deploying Symfony Docker to production with an active Xdebug extension,
-it's recommended to add a custom stage to the end of the `Dockerfile`.
+On Linux, Mac and other Unix-likes:
 
-```Dockerfile
-# Dockerfile
-FROM symfony_php AS symfony_php_debug
-
-ARG XDEBUG_VERSION=3.1.2
-RUN set -eux; \
-	apk add --no-cache --virtual .build-deps $PHPIZE_DEPS; \
-	pecl install xdebug-$XDEBUG_VERSION; \
-	docker-php-ext-enable xdebug; \
-	apk del .build-deps
 ```
-
-## Configure Xdebug with Docker Compose Override
-
-Using an [override](https://docs.docker.com/compose/reference/overview/#specifying-multiple-compose-files) file named `docker-compose.debug.yml` ensures that the production
-configuration remains untouched.
-
-As an example, an override could look like this:
-
-```yaml
-# docker-compose.debug.yml
-version: "3.4"
-
-services:
-  php:
-    build:
-      context: .
-      target: symfony_php_debug
-    environment:
-      # See https://docs.docker.com/docker-for-mac/networking/#i-want-to-connect-from-a-container-to-a-service-on-the-host
-      # See https://github.com/docker/for-linux/issues/264
-      # The `client_host` below may optionally be replaced with `discover_client_host=yes`
-      # Add `start_with_request=yes` to start debug session on each request
-      XDEBUG_CONFIG: >-
-        client_host=host.docker.internal
-      XDEBUG_MODE: debug
-      # This should correspond to the server declared in PHPStorm `Preferences | Languages & Frameworks | PHP | Servers`
-      # Then PHPStorm will use the corresponding path mappings
-      PHP_IDE_CONFIG: serverName=symfony
-    extra_hosts:
-      # Ensure that host.docker.internal is correctly defined on Linux
-      - host.docker.internal:host-gateway
+XDEBUG_MODE=debug docker compose up -d
 ```
 
-Build your image with your fresh new XDebug configuration:
+On Windows:
 
-```console
-docker compose -f docker-compose.yml -f docker-compose.debug.yml build
 ```
-
-Then run:
-
-```console
-docker compose -f docker-compose.yml -f docker-compose.debug.yml up -d
+set XDEBUG_MODE=debug&& docker compose up -d
```

Comment:
Boring Windows stuff again but it might be worth mentioning that this command sets the XDEBUG_MODE env for the terminal session (not only for the current command like on mac/linux). So to turn off debug your have to run 
`set XDEBUG_MODE=off&& docker compose up -d`
or 
`set XDEBUG_MODE=develop&& docker compose up -d`